extension/src/utils.ts

```diff
@@ -1,17 +1,31 @@
-import { window, Selection } from 'vscode';
-import { defaultComponentName } from './constants';
+import { Selection, window } from 'vscode';
+import { defaultComponentName, defaultSkipImport } from './constants';
 
 export async function showInputBox(): Promise<string> {
-    const result = await window.showInputBox({
-        value: '',
-        placeHolder: 'Choose your component name ...',
-    });
-    return result || defaultComponentName;
+  const result = await window.showInputBox({
+    value: '',
+    placeHolder: 'Choose your component name ...',
+  });
+  return result || defaultComponentName;
 }
 
-export function createScript(newComponent: string, activeFile: string, selection: Selection): string {
-    const { start, end } = selection;
-    const script = `ng g @componizer/schematics:ng-componize --name ${newComponent} --activeFile ${activeFile} --start ${start.line} --end ${end.line}`;
-    window.showInformationMessage(script);
-    return script;
-}
\ No newline at end of file
+export async function showInputBoxForSkipImport(): Promise<string> {
```

Comment:
I'm not sure but I believe there is a possibility to have a checkbox with an input box, this will make skipImport more user friendly.
Let's merge and then I'll do the change on my side